src/ui/server/controllers/insurance/export-contract/how-was-the-contract-awarded/index.ts

```diff
@@ -99,6 +101,19 @@ export const post = async (req: Request, res: Response) => {
 
   const validationErrors = generateValidationErrors(payload);
 
+  const sanitised = sanitiseData(payload) as ObjectType;
+
+  /**
+   * Map the payload into an AWARD_METHOD object structure with an id property.
+   * Otherwise, the nunjucks template needs 2x conditions.
+   */
+  const submittedValues = {
```

Comment:
could this go in the if block (feels like wasted resources if only happens if validation errors and not otherwise)

Reply:
Whoops great spot, moved